Dnieh/fix hidden hyperlink toolbar

dnieh opened the pull request:
Basically Quill was automatically hiding the hyperlink toolbar if it detected it below a certain height. Problem is that that height was well within the range of the text, and it was being hidden when it shouldn't have been. Likely an issue with having multiple instances of the quill editor within a page.

Commit:
ZENREACH-555 Fixes hyperlink issue.

Commit:
ZENREACH-555 Built files

bobrafie commented on dist/quill.js:
```diff
@@ -9352,7 +9352,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	      this.root.classList.toggle('ql-out-top', this.root.offsetTop <= 0);
 	      this.root.classList.remove('ql-out-bottom');
-	      this.root.classList.toggle('ql-out-bottom', this.root.offsetTop + this.root.offsetHeight >= this.quill.root.offsetHeight);
+	      this.root.classList.toggle('ql-out-bottom', this.root.offsetTop + this.root.offsetHeight >= this.quill.root.offsetHeight + 100);
```
what if the font size is different

Commit:
ZENREACH-555 Replaces fragile height logic.

dnieh commented:
@bobrafie, I removed the bit of height logic since it's fragile and likely to have unexpected behavior if the font size changes. Instead I changed the styles to not hide the hyperlink popup toolbar as it should always show when a user clicks the hyperlink button.
